packages/react/src/component-tests/IcSideNavigation/IcSideNavigation.cy.tsx

```diff
@@ -348,14 +349,31 @@ describe("IcSideNavigation", () => {
       cy.checkSideNavSize(false);
     });
 
-    it("should render collapsed and expanded when expanded state is externally controlled", () => {
+    it("should render expanded when expanded state is externally controlled", () => {
       mount(<DynamicExpandedSideNav />);
 
       cy.get("#expand-btn").click();
+
+      cy.checkA11yWithWait(undefined, 500);
       cy.checkSideNavSize(true);
+      cy.compareSnapshot({
+        name: "controlled-expanded",
+        testThreshold: setThresholdBasedOnEnv(DEFAULT_TEST_THRESHOLD + 0.024),
```

Comment:
pr looks good. I have a question. Do you have a policy for threshold, or is mainly adjusted to pass the CI with the minimal threshold possible?

Reply:
The [guidance in our CONTRIBUTING.md file](https://github.com/mi6/ic-ui-kit/blob/main/CONTRIBUTING.md#generatingupdating-cypress-screenshots) explains our process for the visual regression tests. We basically generate the screenshots on the CI and then run the tests locally against those screenshots and set the threshold as low as possible (0.01% above the difference).